### DISCUSSION TITLE
Add a new calculation field like "Total Price"

### DISCUSSION THREAD

**Discussion by deepti4666** [YetiForceCompany/YetiForceCRM#14561] [How to]
I would like to add a new field to calculate the net purchase amount which is Qty*Purchase

I have duplicate YetiForceCRM/modules/Vtiger/inventoryfields/TotalPrice.php

It shows up on the add field and detail view - 
![image](https://user-images.githubusercontent.com/57940552/115049271-d47db780-9eda-11eb-8447-e54bc2dca7d0.png)

But when I edit the calculation, it gives me an error 
#0 [error] Unable to load template 'file:modules/Vtiger/inventoryfields/EditViewBase.tpl' in 'modules/Vtiger/Edit/InventoryItem.tpl'
 --> Smarty: Unable to load template 'file:modules/Vtiger/inventoryfields/EditViewBase.tpl' in 'modules/Vtiger/Edit/InventoryItem.tpl' <--

**Comment by vovpff** [comment]
To add new calculated column you need to duplicate a lot. Main function what calculate values implemented on js and you need js experience to achieve this.
